codec/src/test/resources/io/netty/handler/codec/xml/sample-04.xml

```diff
@@ -649,20 +649,19 @@
         </plugin>
         <plugin>
           <artifactId>maven-dependency-plugin</artifactId>
-          <version>2.8</version>
+          <version>2.10</version>
         </plugin>
         <plugin>
           <artifactId>maven-assembly-plugin</artifactId>
-          <version>2.4</version>
+          <version>2.6</version>
         </plugin>
         <plugin>
-          <!-- Do NOT upgrade -->
           <artifactId>maven-jxr-plugin</artifactId>
-          <version>2.2</version>
+          <version>2.5</version>
```

Comment:
Need to check if `mvn package` generates xref files under the `all/target/xref` directory correctly. IIRC, after 2.2, maven-jxr-plugin ran only as a site report plugin which does not meet our needs.